### PR TITLE
[F] HealEntireOrgJob review comment fix (ENT-848)

### PR DESCRIPTION
**Pull request by abhiskum**
@Ceiu Updated code changes as per your suggestions in https://github.com/candlepin/candlepin/pull/2386. 
Please review.

### CODE REVIEW

File: server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java [hunk 1]
```diff
@@ -72,18 +71,23 @@ public HealEntireOrgJob(Entitler entitler, ConsumerCurator consumerCurator, Owne
...
+            if (owner == null) {
+                log.warn("Healing attempted against non-existent org id \"{}\"", ownerKey);
+                throw new JobExecutionException(
+                    i18n.tr("Healing attempted against non-existent org id \"{}\"", ownerKey));
+            }
             if (owner.isAutobindDisabled() || owner.isContentAccessEnabled()) {
                 String caMessage = owner.isContentAccessEnabled() ?
                     " because of the content access mode setting" : "";
-                throw new BadRequestException(
+                throw new JobExecutionException(
```
Comment (Ceiu) [inline server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java]: Remember to set the terminal flag to true here: 

```new JobExecutionException(i18n.tr("..."), true);```

File: server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java [hunk 2]
```diff
@@ -72,18 +71,23 @@ public HealEntireOrgJob(Entitler entitler, ConsumerCurator consumerCurator, Owne
...
 
-            final JobArguments arguments = context.getJobArguments();
-            final String ownerKey = arguments.getAsString(OWNER_KEY);
+            JobArguments arguments = context.getJobArguments();
+            String ownerKey = arguments.getAsString(OWNER_KEY);
             Owner owner = ownerCurator.getByKey(ownerKey);
+            if (owner == null) {
+                log.warn("Healing attempted against non-existent org id \"{}\"", ownerKey);
+                throw new JobExecutionException(
+                    i18n.tr("Healing attempted against non-existent org id \"{}\"", ownerKey));
```
Comment (Ceiu) [inline server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java]: Minor nit: the key and ID are different values, so this error message might be a bit confusing.

Also, need the terminal flag here as well.

File: server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java (hunk 1 above)
Comment (abhiskum) [reply server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java]: Done. I misunderstood your earlier comment.

File: server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java (hunk 2 above)
Comment (abhiskum) [reply server/src/main/java/org/candlepin/async/tasks/HealEntireOrgJob.java]: Done.

**Comment by Ceiu** [comment]
ack

**Pull request merged**